packages/core/tracing/src/locales/en.json

```diff
@@ -0,0 +1,38 @@
+{
+  "header": {
+    "content": "Content of the header",
+    "title": "Title of the header"
+  },
+  "trace_viewer": {
+    "empty_state": {
+      "nothing_to_display": "There's nothing to display for the selected span",
+      "unselected": "Selected span will be shown here"
+    },
+    "span_attributes": {
+      "title": "Span Attributes"
+    },
+    "span_events": {
+      "exception_message": {
+        "title": "Exception message"
```

Comment:
message -> Message?

Reply:
We just heard from the design team yesterday that we would progressively use sentence case across the platform. I have replaced "Span Attributes" with "Span attributes".